Gifski/Gifski.swift

```diff
@@ -94,6 +96,11 @@ final class Gifski {
 			}
 
 			generator.generateCGImagesAsynchronously(forTimePoints: frameForTimes) { result in
+				guard !progress.isCancelled else {
```

Comment:
This is needed as Apple is slow at cancelling when we call `generator.cancelAllCGImageGeneration()`, so we end up with `progress.isCancelled == true` and `result.success`, which ends up making it fail.